src/strategies/erc20-received/index.ts

```diff
@@ -0,0 +1,154 @@
+import fetch from 'cross-fetch';
+import { InfuraProvider, Web3Provider } from '@ethersproject/providers';
+import { formatUnits } from '@ethersproject/units';
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { createDfuseClient } from '@dfuse/client';
```

Comment:
I'd prefer if we don't introduce a new dependency here, that would make the library heavier, i've seen dfuse being used just with fetch here https://github.com/snapshot-labs/snapshot.js/pull/66/files could you use the same approach?

Reply:
No problem! Fixed in https://github.com/snapshot-labs/snapshot.js/pull/60/commits/4fcb966ae78c4cbab780118f5ed39294d945d6b0